test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractMintHTSSuite.java

```diff
@@ -381,9 +384,25 @@ private HapiApiSpec transferNftAfterNestedMint() {
 				).then(
 						getTxnRecord(nestedTransferTxn).andAllChildRecords().logged(),
 						childRecordsCheck(nestedTransferTxn, SUCCESS,
-								recordWith().status(SUCCESS),
-								recordWith().status(SUCCESS).tokenTransfers(NonFungibleTransfers.changingNFTBalances().
-												including(nonFungibleToken, TOKEN_TREASURY, theRecipient, 1)))
+								recordWith()
+										.status(SUCCESS)
+										.contractCallResult(
+												resultWith()
+														.gasUsed(expectedGasUsage)
+														.contractCallResult(functionResult()
```

Comment:
I'm not liking how this reads.  It implies a general function result for a contract call but in reality it is the HTSPrecompile function result.  Can the static method be renamed to reflect we are generating a HTS precompile function result for comparison?  such as `htsPrecompileResult()`?